tools/summaries/build_ranking_based_summaries.py

```diff
@@ -205,12 +183,29 @@ def build_ranking_based_summaries(place_type: str, parent_place_dcid: str,
     json.dump(summaries, out_file, indent=4)
 
 
-def main():
-  build_ranking_based_summaries(place_type="State",
-                                parent_place_dcid="country/USA",
-                                stat_var_json=_STAT_VAR_JSON,
-                                output_file=_OUTPUT_FILENAME)
+@click.command()
+@click.argument('place_type', help="place type to generate summaries for")
+@click.argument(
+    'parent_place_dcid',
+    help="containing parent place of places to generate summaries for")
+@click.argument('stat_var_json',
+                default=_STAT_VAR_JSON,
+                help="path to stat var config file")
+@click.argument('output_file',
+                default=_OUTPUT_FILE,
+                help="path to write summaries to")
+def main(place_type: str,
+         parent_place_dcid: str,
+         stat_var_json: str = _STAT_VAR_JSON,
+         output_file: str = _OUTPUT_FILE):
+  build_ranking_based_summaries(place_type=place_type,
+                                parent_place_dcid=parent_place_dcid,
+                                stat_var_json=stat_var_json,
+                                output_file=output_file)
 
 
 if __name__ == "__main__":
-  main()
+  main(place_type="State",
```

Comment:
Instead of setting the params here, just call `main()` and rely on the `@click.option(default=...)` for default values

Reply:
Removed args from `main()` in `__main__`